src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -472,6 +472,13 @@ public enum ViewAccessibilityMode
             /// </summary>
             [EditorBrowsable(EditorBrowsableState.Never)]
             Custom,
+            /// <summary>
+            /// Ignore all accessibility implementation. Cannot use View.Accessibility...()
+            /// will be necessary to reduce memory of the View.
+            /// Note that it will make invalidate it's all childrens and parent's accessibility features.
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            Ignore,
```

Comment:
This is `View` class, so I'm wondering whether we should add `Accessibility` naming on this feature because it's only related to accessibility.
Furthermore, there is a function that makes disabled AT-SPI itself in `Accessibility` class of NUI :
https://github.com/Samsung/TizenFX/blob/main/src/Tizen.NUI/src/public/Accessibility/Accessibility.cs#L173
We shared one to other developers in order to use this API, so I think we can use this one probably?
If not, it would be better to think the naming again :)

Reply:
Hello world!

`Accessibility.BridgeDisableAutoInit()` is disable whole systemical AT-SPI system, not for specific `View`.
Also that API didn't ignore the memory allocation for control's accesibility relative items.

But also, I got some review that this creation feature looks equavalance with `View.AccessibilityHidden = true` for user side. So I need to refactorize behavior soon :(